experimental/regression_suite/shark-test-suite-models/llama/test_llama.py

```diff
@@ -0,0 +1,77 @@
+# Copyright 2024 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+import pytest
+from ireers_tools import *
+import os
+from conftest import VmfbManager
+from pathlib import Path
+
+rocm_chip = os.getenv("ROCM_CHIP", default="gfx90a")
+vmfb_dir = os.getenv("TEST_OUTPUT_ARTIFACTS", default=Path.cwd())
+
+###############################################################################
+# Fixtures
+###############################################################################
+llama_mlir = fetch_source_fixture(
+    "https://sharkpublic.blob.core.windows.net/sharkpublic/halo-models/llm-dev/llama3_8b/8b_f16_decomposed_11_22.mlir",
+    group="llama_fp16_8b",
+)
```

Comment:
Can you add a comment here indicating how this was generated? What version of the packages were used, what command were run, etc.

Ideally link to some documentation in https://github.com/nod-ai/shark-ai/tree/main/docs

See the earlier comment on https://github.com/iree-org/iree/pull/14915#discussion_r1316401801

> We ultimately want all of these coming from common places like that which we update with a generation script. However, as long as we're in this phase with this one of it being a bit hand-massaged, I'm ok with Dan taking responsibility to keep it updated.

;)

(the tests that PR updated are also linked in my other comment... and have been outdated + disabled for months. IDK how to update them since that isn't documented)